aten/src/ATen/ops/from_blob.h

```diff
@@ -85,6 +91,7 @@ class TORCH_API TensorMaker {
   std::unique_ptr<void, ContextDeleter> ctx_{nullptr, detail::noopDelete};
   c10::optional<Device> device_{};
   TensorOptions opts_{};
+  bool resizeable_{}; // Allows the storage of this tensor to be resized
```

Comment:
Since you don't store the allocator here, basically this only can be used when you are meta, right?

Reply:
yep. This probably deserves a comment - I'll add one.